docs/17-common-problems-and-their-solutions.md

```diff
@@ -4,97 +4,22 @@ Common problems and their solutions
-This can be achieved by deleting the table and then creating it again, but the following steps may well be quicker:
-
-You can reinitialise the table quickly by running the following command - note that reinitialising a table will delete
-all data in the table. If you want to change the table schema make sure you change
-the schema in the table properties before running this script:
-
-```bash
-./scripts/utility/reinitialiseTable.sh <instance-id> <table-name> <optional-delete-partitions-true-or-false> <optional-split-points-file-location> <optional-split-points-file-base64-encoded-true-or-false>
-```
-
-e.g.
-
-```bash
-./scripts/utility/reinitialiseTable.sh my-sleeper-config-bucket my-sleeper-table true /tmp/split-points.txt false
-```
-
-Alternatively you can use a more manual approach. This may be better if you want to significantly change the table
-properties.
-
-For a table with a Dynamo DB state store:
-
-- Delete all the files `partition*/*` in the S3 bucket for the table (the bucket will be named
-  `sleeper-<instance-id>-table-<table-name>`). This can be done either in the AWS console or using the CLI.
-- Delete all the entries in the DynamoDB state store tables: `sleeper-<instance-id>-table-<table-name>-active-files`,
-  `sleeper-<instance-id>-table-<table-name>-gc-files`, `sleeper-<instance-id>-table-<table-name>-partitions`.
-- If you want to change the schema of the table, then edit the table properties file in the config bucket in S3.
-- Reinitialise the state store:
-
-```bash
-java -cp scripts/jars/clients-*-utility.jar sleeper.statestore.InitialiseStateStore <instance-id> <table-name>
-```
-
-For a table with an S3 state store:
-
-- Delete all the files `partition*/*` in the S3 bucket for the table (the bucket will be named
-  `sleeper-<instance-id>-table-<table-name>`). This can be done either in the AWS console or using the CLI.
-- Delete all the files `statestore/*` in the same S3 bucket for the table.
-- Delete all the entries in the DynamoDB table: `sleeper-<instance-id>-table-<table-name>-revisions`,
-- If you want to change the schema of the table, then edit the table properties file in the config bucket in S3.
-- Reinitialise the state store:
-
-```bash
-java -cp scripts/jars/clients-*-utility.jar sleeper.statestore.InitialiseStateStore <instance-id> <table-name>
-```
-
-## Why is adding a table to an existing Sleeper instance so slow?
-
-Each table stores its data in its own bucket, and has its own state store (by default the DynamoDB-based state store
-is used which uses 3 DynamoDB tables). When a new Sleeper table is created, these resources all need to be created
-and then the various components of Sleeper (lambda, ECS containers, etc) all need to have the permissions of their
-roles updated so that they can interact with these resources. This is done using cdk and it takes several minutes.
+(Presently the implemenation is based on arrow ingest whereas previously it was based on an arraylist.)
```

Comment:
I think the reason to mention this would be to note that this works differently if you use an array list? I think the brackets aren't needed here.

Reply:
Addressed